sky/backends/backend_utils.py

```diff
@@ -2017,16 +2016,15 @@ def refresh_cluster_record(
     Args:
```

Comment:
(styleguide nit): if 2 spaces, reduce gap to "Args" from 4 to 2 too

Reply:
This is the hanging line of the previous line, it seems either 2 or 4 works for it according to the google style guide: https://google.github.io/styleguide/pyguide.html#doc-function-args

We use 2 space to make it align with the later fields in the docstr.  :)